pkg/kf/commands/routes/integration_test.go

```diff
@@ -29,32 +30,60 @@ import (
 func TestIntegration_Routes(t *testing.T) {
 	RunKfTest(t, func(ctx context.Context, t *testing.T, kf *Kf) {
 		hostname := fmt.Sprintf("some-host-%d", time.Now().UnixNano())
+		domain := "example.com"
+		path := "some-path"
 
-		findRoute := func(shouldFind bool) {
-			RetryOnPanic(ctx, t, func() {
-				var found bool
-				for _, line := range kf.Routes(ctx) {
-					expected := hostname + " example.com /some-path"
-					actual := strings.Join(strings.Fields(line), " ")
-					if expected == actual {
-						found = true
-						break
-					}
-				}
-
-				if shouldFind != found {
-					// We'll panic so we can use our retry logic
-					panic(fmt.Errorf("Wanted %v, got %v", shouldFind, found))
-				}
-			})
-		}
+		// TODO: use the domain from the cluster.
```

Comment:
Can we attach an issue to this TODO?

Reply:
this actually wasn't necessary for the test, so removed the todo